variables.tf

```diff
@@ -3,6 +3,12 @@ variable "account" {
   description = "The name of the account"
 }
 
+variable "create_email_address" {
```

Comment:
Also think this should be `create_email_alias`, since that's what it's creating and matches the resource.